composer.json

```diff
@@ -23,5 +26,10 @@
         "branch-alias": {
             "dev-master": "1.1-dev"
         }
+    },
+    "config": {
+        "allow-plugins": {
+            "dealerdirect/phpcodesniffer-composer-installer": true
+        }
```

Comment:
Ok. I'll clone this later tonight and try to run `phpcs` without this plugin. If all goes well (and Slevomat rules are checked), I'll see if this can be removed. Otherwise, I'll keep it in.

Either way, I'll expect to have this PR merged Thursday. Thanks again!

Reply:
Testing out the PR locally, I noticed this wasn't required to run `phpcs`; however, I read up further on the Composer 2.2 changes and the [PHPCSStandards/composer-installer README](https://github.com/PHPCSStandards/composer-installer#usage) and now understand this is a forward-thinking change to ensure Slevomat is properly installed _after_ the new security rules take effect in July.

LGTM.